README.md

```diff
@@ -0,0 +1,26 @@
+## Kasse
+
+This is an Implementation of a payment system for the
+[Heidelberg Chaostreff](https://www.noname-ev.de). It is currently in a basic
+stage of devellopment, so this README.md should give you enough information to
```

Comment:
"so this README.md should give you enough information to install it and start testing" is a weird implication in my mind. Maybe there is an "only" missing here?